tests/frontend/test_kibana_app/cypress/integration/features/management/settings/about.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Wazuh version information
+
+  As a kibana user
+  I want to check the about information
+  in order to see information about the system
```

Comment:
`about the app` or plugin (?)

Reply:
about the app, or at least that is what the page displays.

nowadays we are just validating all of these elements are shown on the page
![Selection_134](https://user-images.githubusercontent.com/23361539/122924418-20663600-d33c-11eb-8216-24388e768437.png)

but the idea is validating we have the correct version when we need to deploy one version in particular, in the future